### ISSUE TITLE
"please report this issue"...

### ISSUE THREAD

**Issue by pdbruyn** [yt-dlp/yt-dlp#7845]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Netherlands

### Provide a description that is worded well enough to be understood

Tried to download a youtube video wich failed, was asked to report the issue

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
PS G:\youtube-dl> .\yt-dlp.exe https://www.youtube.com/watch?v=jtRrhL9r92w -vU
[debug] Command-line config: ['https://www.youtube.com/watch?v=jtRrhL9r92w', '-vU']
[debug] User config "C:\Users\Peter\AppData\Roaming\yt-dlp\config.txt": ['--no-mtime', '--merge-output-format', 'mp4']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-108915-g5661c8715c-20221102 (setts), ffprobe N-108915-g5661c8715c-20221102
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[youtube] Extracting URL: https://www.youtube.com/watch?v=jtRrhL9r92w
[youtube] jtRrhL9r92w: Downloading webpage
[youtube] jtRrhL9r92w: Downloading ios player API JSON
[youtube] jtRrhL9r92w: Downloading android player API JSON
[youtube] jtRrhL9r92w: Downloading m3u8 information
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] jtRrhL9r92w: Downloading 1 format(s): 244+251
[download] Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].mp4 has already been downloaded
PS G:\youtube-dl>
```

**Comment by bashonly** [comment]
There is no error in the log?

**Comment by pdbruyn** [comment]
it gave an error the first time, but I guess it downloaded anyway. I deleted the download and tried it again:

PS G:\youtube-dl> .\yt-dlp.exe https://www.youtube.com/watch?v=jtRrhL9r92w -vU
[debug] Command-line config: ['https://www.youtube.com/watch?v=jtRrhL9r92w', '-vU']
[debug] User config "C:\Users\Peter\AppData\Roaming\yt-dlp\config.txt": ['--no-mtime', '--merge-output-format', 'mp4']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-108915-g5661c8715c-20221102 (setts), ffprobe N-108915-g5661c8715c-20221102
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[youtube] Extracting URL: https://www.youtube.com/watch?v=jtRrhL9r92w
[youtube] jtRrhL9r92w: Downloading webpage
WARNING: [youtube] unable to extract initial player response; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[youtube] jtRrhL9r92w: Downloading ios player API JSON
[youtube] jtRrhL9r92w: Downloading android player API JSON
[youtube] jtRrhL9r92w: Downloading iframe API JS
[youtube] jtRrhL9r92w: Downloading player 3cd2d050
[youtube] jtRrhL9r92w: Downloading web player API JSON
[youtube] jtRrhL9r92w: Downloading m3u8 information
WARNING: [youtube] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube] Incomplete data received in embedded initial data; re-fetching using API.
[youtube] jtRrhL9r92w: Downloading initial data API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] jtRrhL9r92w: Downloading 1 format(s): 244+251
[debug] Invoking http downloader on "https://rr3---sn-32o-bg9e.googlevideo.com/videoplayback?expire=1692138648&ei=OKjbZNn_GIvD1gL3sKMY&ip=2a02%3Aa471%3Aa103%3A1%3Aa5f1%3A4e65%3A42ab%3Ac898&id=o-AC7aPQJyGyyRLlfRJ3CYXHVzI342A8HErSte_mU1Nf1v&itag=244&source=youtube&requiressl=yes&mh=qt&mm=31%2C29&mn=sn-32o-bg9e%2Csn-32o-5hn6&ms=au%2Crdu&mv=m&mvi=3&pl=41&initcwndbps=2098750&vprv=1&svpuc=1&mime=video%2Fwebm&gir=yes&clen=8231303&dur=216.000&lmt=1685393197446146&mt=1692116496&fvip=3&keepalive=yes&fexp=24007246%2C51000024&c=IOS&txp=5437434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAKTwGvdk8RWLeRM387wJSdajZsSQ7Ah4C_Nr8Aknk58-AiEA4BAt21HUCGaFuaEjiE5QeCC3ZqKCGpUDgdxtgU37r64%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgTZUidMVJPSGw8US8leIsN0aCYdBwEAao0ktqY1VlcHICIGHbOh9bCmk9b86rOOspeXgPPiuXlzY692WLB99mF_bh"
[debug] File locking is not supported. Proceeding without locking
[download] Destination: Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].f244.webm
[download] 100% of    7.85MiB in 00:00:01 at 5.39MiB/s
[debug] Invoking http downloader on "https://rr3---sn-32o-bg9e.googlevideo.com/videoplayback?expire=1692138648&ei=OKjbZM3nIr6v6dsP5o6G-AY&ip=2a02%3Aa471%3Aa103%3A1%3Aa5f1%3A4e65%3A42ab%3Ac898&id=o-AMFrbS0FifCV_vl8phsJbwCgX0oAU7f7WaxcAJck8e3M&itag=251&source=youtube&requiressl=yes&mh=qt&mm=31%2C29&mn=sn-32o-bg9e%2Csn-32o-5hn6&ms=au%2Crdu&mv=m&mvi=3&pl=41&initcwndbps=2098750&spc=UWF9f_N00dbm5nbRgW8x85Jk5mwILzw&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=3569409&dur=216.041&lmt=1685391135669200&mt=1692116496&fvip=3&keepalive=yes&fexp=24007246%2C24363391&c=ANDROID&txp=5432434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhALRK0gExYIUQC7XAhdFJ9bWT9tjdRTYZK3l-bCZGZgdpAiBYLkCAt1ssHzzSeaMmsh4pf5rirYeMcCODr_La5CXVEw%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgTmtB6YAd5cRi9b6GnFOMwOpdTKa0hMwH33VTL9fS5jUCIQDFsm_qtCP6YPfCkQilAHot4veNwatJIMFU5KAz-DfIWg%3D%3D"
[download] Destination: Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].f251.webm
[download] 100% of    3.40MiB in 00:00:00 at 4.64MiB/s
[Merger] Merging formats into "Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].f244.webm" -i "file:Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].f251.webm" -c copy -map "0:v:0" -map "1:a:0" -movflags "+faststart" "file:Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].temp.mp4"
Deleting original file Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].f251.webm (pass -k to keep)
Deleting original file Alphaville     --      Forever    Young      Live   Video   HQ [jtRrhL9r92w].f244.webm (pass -k to keep)
PS G:\youtube-dl>

**Comment by bashonly** [comment]
Ah I see. This was fixed by 378ae9f9fb8e8c86e6ac89c4c5b815b48ce93620. You can run `yt-dlp --update-to nightly` if you want the patched version

Closing as duplicate of #7594
